main/lib/plugins/CaliceGenericConverterPlugin.cc

```diff
@@ -191,25 +217,28 @@ namespace eudaq {
 	//  cout << "collection added." << endl;
       }
       col->parameters().setValue("DataDescription", dataDesc);
-      //add timestamp (EUDAQ!!)
+      //add timestamp (set by the Producer, is EUDAQ, not real timestamp!!)
       struct tm *tms = localtime(&timestamp);
       char tmc[256];
       strftime(tmc, 256, "%a, %d %b %Y %T %z", tms);
```

Comment:
Check the spaces --> %b and following are shown in red, might by a special character problem.